### PR TITLE
feat: task 3, lambda API integration

### PR DESCRIPTION
**Pull request by dzianis-l**
Task 3: FE integration of lambda and Gateway API

### CODE REVIEW

File: src/mocks/handlers.ts [hunk 1]
```diff
@@ -6,9 +6,6 @@ import { Order } from "~/models/Order";
 import { AvailableProduct, Product } from "~/models/Product";
 
 export const handlers = [
-  rest.get(`${API_PATHS.bff}/product`, (req, res, ctx) => {
```
Comment (AL1088717) [inline src/mocks/handlers.ts:9]: You can keep this handler in mocks, cause it's only there for local development (check line 17 in src/index) . It's up to you, however.

**Comment by AL1088717** [comment]
@dzianis-l
/product/{productId} endpoint fails with CORS
image

File: src/mocks/handlers.ts (hunk 1 above)
Comment (dzianis-l) [reply src/mocks/handlers.ts:9]: sure, I'd rather keep it, thanks!

**Comment by dzianis-l** [comment]
> @dzianis-l /product/{productId} endpoint fails with CORS (you can check it if you try to manage product in the list) image

Thanks for the comment.

According to evaluation criteria "[Task 3.2 bullet point 4](https://github.com/EPAM-JS-Competency-center/cloud-development-course-initial/blob/main/3_serverless_api/task.md#task-32) This endpoint is not needed to be integrated with Frontend right now."

Thus no necessity to do it right now.
However, I've integrated it.

Thanks

**Pull request merged**